### DISCUSSION TITLE
settings.ini / .reg file maybe?

### DISCUSSION THREAD

**Discussion by ShiroUwUs7** [valinet/ExplorerPatcher#147] [General]
Was wondering if there would be a settings file that it would generate to keep it for backups due to using it in conjunction with startallback to fit the needs of alt tab, context menus dynamic transparency etc.  basically combining the two for feature sets sense there both extremally light weight. and have no marginable performance impact from what i've tested.

also is there a manual file link to get back to the preferences? I can't seem to get back to it once I disable the win10 start menu. even if i disable startallback.  (edit: found the shortcut from the old readme: `Rundll32 %windir%\dxgi.dll,ZZGUI` )

only other thing I cant really figure out is: going back to the win11 "ribbon" instead of the win 10 version: 
![P8Oil2a 1](https://user-images.githubusercontent.com/53096166/137647716-2c1ddab4-2b03-4290-8416-3e6dcfda99bc.png)

I'll attach some screenshots:
![Untitled](https://user-images.githubusercontent.com/53096166/137647376-1375f94a-fb33-41d0-860e-7941cc478420.png)
![Untitled1](https://user-images.githubusercontent.com/53096166/137647378-df6696cc-c8d5-4e64-b6a7-3d00265097b3.png)
![Untitled2](https://user-images.githubusercontent.com/53096166/137647379-086d6efb-bf8d-4877-a43b-06edcc130297.png)
![Untitled3](https://user-images.githubusercontent.com/53096166/137647380-e5418a31-5f59-4755-904e-bf4cc43478d3.png)
![Untitled4](https://user-images.githubusercontent.com/53096166/137647381-87616595-03ed-408c-a55a-7f592b23ebf6.png)

**Comment by ShiroUwUs7** [comment]
Also I can't seem to find where to enable these: 
![ah6ctN6 1](https://user-images.githubusercontent.com/53096166/137655847-18948815-01c2-40df-b043-545f1ffe3099.png)

![FkWvkjD 1](https://user-images.githubusercontent.com/53096166/137656054-6984e4e2-020e-458b-b345-aeae3f76d760.png)

**Reply by valinet** [reply]
> 
> 
> Also I can't seem to find where to enable these: ![ah6ctN6 1](https://user-images.githubusercontent.com/53096166/137655847-18948815-01c2-40df-b043-545f1ffe3099.png)
> 
> ![FkWvkjD 1](https://user-images.githubusercontent.com/53096166/137656054-6984e4e2-020e-458b-b345-aeae3f76d760.png)

![image](https://user-images.githubusercontent.com/6503598/137668624-ccbcbe18-c6bd-4e2e-a5e2-14338854fc4a.png)

**Reply by ShiroUwUs7** [reply]
oh I see... Sorry kind of new to win 11. Do you happen to know if ExplorerPatcher has a general reg key / ini / config file? That I can save etc.

**Reply by valinet** [reply]
Consult https://github.com/valinet/ExplorerPatcher/blob/master/ExplorerPatcher/settings.reg. Settings are saved in general in the original place where it historically is, or otherwise in `HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher`.

**Reply by ShiroUwUs7** [reply]
Thank you! I'll take a look at it. and the only last question I really had was reverting back to win 11 explorer.exe ribbon if possible. 
as I didn't see anything that hinted at it in the settings.  unless 
```
;T File Explorer
[-HKEY_CURRENT_USER\Software\Classes\CLSID\{d93ed569-3b3e-4bff-8355-3c44f6a52bb5}\InprocServer32]
``` 
is what I'm looking for. But didn't find anything hinting at it in the actual preferences.

**Reply by valinet** [reply]
It's there: "Disable Windows 11 command bar" in "File Explorer" settings in the Properties GUI.

**Reply by ShiroUwUs7** [reply]
oh. I'm blind... Sorry.
i'll def take a look in the registry once I set everything up again.

**Reply by DRKiarsis** [reply]
its not there "Disable Windows 11 command bar" in "File Explorer" settings in the Properties GUI. in version 22621.1555.55.1

I cannot find how to enable the win 10 ribbon instead of win 11
